website-next/components/menu.js

```diff
@@ -28,6 +28,9 @@ export function Menu() {
                   key={item.name}
                   href={item.href}
                   className='hidden md:block font-medium text-gray-900 hover:text-gray-900 text-lg'
+                  onMouseDown={() =>
+                    gtag('event', 'navigate', { category: 'links', label: item.href, value: 1 })
```

Comment:
The docs give the impression that a non-negative integer `value` field is required, hence the `1` here. I've checked and we can see this event in the GA dashboard.